src/uu/seq/src/seq.rs

```diff
@@ -197,25 +209,38 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         crash_if_err!(1, slice.parse())
     };
 
+    let is_negative_zero_f64 = |x: f64| x == -0.0 && x.is_sign_negative() && largest_dec == 0;
     let result = match (first, last, increment) {
-        (Number::MinusZero, Number::BigInt(last), Number::BigInt(increment)) => print_seq_integers(
-            (BigInt::zero(), increment, last),
+        // For example, `seq -0 1 2` or `seq -0 1 2.0`.
```

Comment:
This handles the case where -0 is given as an integer.

Reply:
can you add a test with this ?  thanks